### DISCUSSION TITLE
Can I join the game development？

### DISCUSSION THREAD

**Discussion by chenpishui12345** [HasangerGames/suroi#281] [General]
Can I join the game development？Hope to approve

**Comment by 1092384** [comment]
you joined github 40 minutes ago

**Reply by chenpishui12345** [reply]
yes

**Comment by 1092384** [comment]
who are you

**Reply by chenpishui12345** [reply]
China

**Reply by chenpishui12345** [reply]
you can call me cps

**Comment by DamienVesper** [comment]
You are welcome to make any PRs to the public repository for bug fixes or features as you feel necessary. However, push access to the repo, and access to development and staging branches are limited.

**Reply by chenpishui12345** [reply]
how can I join it?

**Comment by Katloo24** [comment]
@chenpishui12345 Please join our Discord for easier communication.

**Reply by chenpishui12345** [reply]
ok

**Reply by chenpishui12345** [reply]
how to join it

**Reply by DamienVesper** [reply]
Please join our Discord for easier communication. https://discord.com/invite/j8pPtMa843

**Reply by chenpishui12345** [reply]
Discord cannot be used in China as it is prohibited

**Reply by 1092384** [reply]
then get a vpn

**Reply by Compositr** [reply]
VERY BAD! DO NOT MENTION THE DISCORD (不和谐聊天应用程序) CHAT APP! 20 social credits have been deducted 低等公民 and your internet access card 上网通行证 has been suspended for: [24 Hours]. Please refrain from mentioning events that never happened that could discredit the great 人民共产党 People’s Communist Party again or we will be forced to 饿了就睡觉 send party agents to escort you to a re-education van [人民行刑车].

**Reply by Compositr** [reply]
We regret to inform you that your family has been taken into re-education camps in Xinjiang. Your social credit score will not be affected by this. Glory to China long live
Xi Jinping 天安门广场抗议 黑人使我不舒服
